spec/models/billing_entity_spec.rb

```diff
@@ -9,13 +9,19 @@
 
   it { is_expected.to belong_to(:organization) }
 
+  it { is_expected.to have_many(:customers) }
+  it { is_expected.to have_many(:invoices) }
+  it { is_expected.to have_many(:invoice_custom_section_selections) }
+  it { is_expected.to have_many(:selected_invoice_custom_sections) }
+  it { is_expected.to have_many(:fees) }
+
```

Comment:
it would be great if also add the other relationships here...

```  
  has_many :subscriptions, through: :customers
  has_many :wallets, through: :customers
  has_many :wallet_transactions, through: :wallets
  has_many :credit_notes, through: :invoices

  has_one :applied_dunning_campaign, class_name: "DunningCampaign"
```